src/Solr/Reindex/Handlers/SolrReindexBase.php

```diff
@@ -99,6 +100,8 @@ protected function processVariant(
         $batchSize,
         $taskName
     ) {
+        // Get current state
+        $originalState = SearchVariant::current_state();
```

Comment:
These `$originalState` commits are from a different PR and showed up after I rebased on to `3`, was originally targetted `3.6` I think what happened is there was a missing merge-up

Reply:
Hmm, we'll want to tidy this up before merging. You get the fun task of counting how many commits you've made and running `git rebase --onto=3 HEAD~x` 😅

Reply:
ok sorted, thanks for the merge-up @dhensby